input/fsh/OD_Operations.fsh

```diff
@@ -1,17 +1,18 @@
-Instance: GetPatientBundleOperation
+Instance: Operation-mcode-patient-everything
 InstanceOf: OperationDefinition
-Description: "Gets an [mCODE Patient Bundle](StructureDefinition-mcode-patient-bundle.html) for a specific patient."
+Description: "Gets an [mCODE Patient Bundle](StructureDefinition-mcode-patient-bundle.html) for a specific patient that contains all of that patient's resources that conform to mCODE Profiles."
 Usage: #definition
 
-* id = "mcode-get-patient-bundle"
-* name = "GetPatientBundle"
-* title = "Retrieve an mCODE Patient Bundle"
```

Comment:
Was the title element intentionally deleted?

Reply:
Thanks, fixing this.